Formula/mold.rb

```diff
@@ -15,6 +19,7 @@ class Mold < Formula
     sha256               x86_64_linux:   "25d98acdaf7133398a274037d6bae48b1624f357f7a05b1893acd8c868a5c747"
   end
 
+  depends_on "python@3.10" => :build # needed to run update-git-hash.py
```

Comment:
Do `uses_from_macos "python" => :build, since: :catalina` instead.

Though I'd much rather not need Python at all -- can we ask upstream if they can provide a way to build without it?

Alternatively, we can probably mock the output of `update-git-hash.py`. It seems a bit much to need Python only to produce a line that says there is no Git hash.

Reply:
The script outputs the following content to `out/git-hash.cc`:
```
#include <string>
namespace mold {
std::string mold_git_hash = "";
}
```

Reply:
Yes, that's what I meant. We can just write that file directly to avoid needing Python. But it would be good to ask upstream if they'll provide a supported way of not having Python available during the build.

Reply:
OK, I'll ask the upstream about that.